app/scripts/views/mixins/password-mixin.js

```diff
@@ -25,12 +25,21 @@ define(function (require, exports, module) {
 
     setPasswordVisibilityFromButton: function (button) {
       var isVisible = this.$(button).is(':checked');
-      this.setPasswordVisibility(isVisible);
+      var targets = this.getTargets(button);
+      this.setPasswordVisibility(isVisible, targets);
     },
 
-    setPasswordVisibility: function (isVisible) {
+    getTargets: function (button) {
+      var unsynchronizedInput = !! this.$(button).attr('data-unsynchronized-input');
+      var passwordField = this.$('.password');
+      if (unsynchronizedInput) {
```

Comment:
no need to create a new `var` for `unsynchronizedInput`, you can just write it out here. It will be shorter if the `.data(...)` approach works.